src/chapters/data-and-variables/chapter-4-exercises.rst

```diff
@@ -0,0 +1,35 @@
+Chapter 4 Exercises
+===================
+
+For the following exercises, you will use the information given below about your space shuttle.
+
+.. list-table:: Space Shuttle Information
+   :widths: auto
+   :header-rows: 1
+
+   * - Data
+     - Value
+   * - Name of the space shuttle
+     - Determination
+   * - Orbit Speed (mph)
+     - 17,500
+   * - G Force at Launch
+     - 3G
+   * - Launch Status
+     - ready for launch
+   * - Distance to Mars (km)
+     - 225,000,000
+   * - Distance to the Moon (km)
+     - 384,400
+
+#. Assign a variable named spaceShuttleName the value of the name of the space shuttle.
+
+#. For each value in the table, if you were to assign them to variables in Javascript, what would be the data type?
+
+#. Time to calculate some important values for our space shuttle!
+
+   #. How long will it take to reach Mars?
```

Comment:
More direction would probably be better here.
- How do we want them to calculate these?
- What are the formulas/algorithms to use?
- Do we need to give them more variables?